WooCommerce/Classes/ViewRelated/Orders/Cells/OrderNoteTableViewCell.swift

```diff
@@ -80,25 +76,26 @@ private extension OrderNoteTableViewCell {
     /// Updates the Status Label + Icon's Color
     ///
     func noteTypeWasUpdated() {
+        let theAuthor = author ?? ""
+        let dateOfCreation = dateCreated ?? ""
         if isCustomerNote {
             iconButton.backgroundColor = StyleManager.statusPrimaryBoldColor
-            statusLabel.text = NSLocalizedString("Note to customer", comment: "Labels an order note to let user know it's visible to the customer")
+            statusLabel.text = NSLocalizedString("\(dateOfCreation) - \(theAuthor) (To Customer)", comment: "Labels an order note to let user know it's visible to the customer")
```

Comment:
Would you consider appending something like this to the comment? 
`Reads like 05:30 PM - woo testing (Private)`

That would make it more clear for translators